README.md

```diff
@@ -1,50 +1,163 @@
-# The Lassie Lounge URL Shortener 
 
-Hacktoberfest Project
+<div align="center">
+  <img src="https://raw.githubusercontent.com/webdev-20/shorti-url-shortener/2a16fa1e8d6a13c4e6aacd5ebd517ed1b6795001/client/public/images/shorti.svg">
+  <h3 align="center">
+      <a href="https://shorti.onrender.com" class="default">View Demo</a> · 
+      <a href="https://github.com/webdev-20/shorti-url-shortener/issues" class="default">Report Bug</a> · 
+      <a href="https://github.com/webdev-20/shorti-url-shortener/issues" class="default">Request Feature</a>
+  </h3>
+  <div>
 
-# The project
+  <a href="">[![Issues](https://img.shields.io/github/issues/webdev-20/shorti-url-shortener.svg?style=for-the-badge)](https://github.com/webdev-20/shorti-url-shortener/issues)</a>
+  <a href="">[![contributors](https://img.shields.io/github/contributors/webdev-20/shorti-url-shortener.svg?style=for-the-badge)](https://github.com/webdev-20/shorti-url-shortener/contributors)</a>
+  <a href="">[![Forked](https://img.shields.io/github/forks/webdev-20/shorti-url-shortener.svg?style=for-the-badge)](https://github.com/webdev-20/shorti-url-shortener)</a>
 
-URL shortener 
+  </div>
+</div>
 
-## Tech Stack
+<p align="center"; style="font-size:115%;">This project helps user to generate a short URL making it easy to share and use.
+</p>
 
-- React
-- CSS Modules
-- Mongo
-- Express
+<a name = "readme-top"></a>
 
-## Features 
-please check the pinned issue
+## Table of Contents
+  
+  1. [About The Project](#about-the-project)
+     - [Built With](#built-with)
+  2. [Getting Started](#getting-started)
+       - [Prerequisites](#prerequisites)
+       - [Run project locally](#run-project-locally)
+  3. [Contributing](#contributing)
+  4. [API Documentation](#api-documentation)
+  5. [Features](#features)
+  6. [Live Site](#live-site)
+  7. [License](#license)
+  8. [Contacts](#contacts)
 
-## Live site
+## About The Project
+
+<blockquote> 
+Users can shorten a link using this app which returns a short code that redirects to the original link. <br>
+
+There are multiple endpoints created under **Authentication** and **Links** tag.<br>
```

Comment:
this line can be removed. I would put some screenshots here but we can probably wait till the page is completed

Reply:
`There are multiple endpoints created under **Authentication** and **Links** tag.` this line is removed now.